### PR TITLE
Refactor layout to align things properly, add a close button.

### PR DESCRIPTION
**Pull request by miguelandres**

### CODE REVIEW

File: remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/view/RemixerFragment.java [hunk 1]
```diff
@@ -146,7 +143,16 @@ public View onCreateView(LayoutInflater inflater,
                            ViewGroup container,
                            Bundle savedInstanceState) {
     View view = inflater.inflate(R.layout.fragment_remixer_list, container, false);
-
+    ImageView closeButton = (ImageView) view.findViewById(R.id.closeButton);
+    closeButton.setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View view) {
+        FragmentManager manager = getFragmentManager();
```
Comment (pingpongboss) [inline remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/view/RemixerFragment.java]: nit: inline

File: remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/view/RemixerFragment.java [hunk 2]
```diff
@@ -146,7 +143,16 @@ public View onCreateView(LayoutInflater inflater,
...
+        FragmentTransaction transaction = manager.beginTransaction();
+        transaction.remove(RemixerFragment.this);
+        transaction.commit();
```
Comment (pingpongboss) [inline remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/view/RemixerFragment.java]: nit: chain

File: remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/widget/BooleanVariableWidget.java [hunk 3]
```diff
@@ -32,7 +33,7 @@
  * Displays an {@link Variable &lt;Boolean&gt;} as a switch.
  */
 @Keep
-public class BooleanVariableWidget extends RelativeLayout
+public class BooleanVariableWidget extends LinearLayout
```
Comment (pingpongboss) [inline remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/widget/BooleanVariableWidget.java:36]: suggest you explicitly set the orientation in xml/java. Suggest java.

**Review by pingpongboss — APPROVED** [review]

File: remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/widget/BooleanVariableWidget.java (hunk 3 above)
Comment (miguelandres) [reply remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/widget/BooleanVariableWidget.java:36]: I think it's harder to read in Java, but agreed. I'll put it on xml though.

File: remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/view/RemixerFragment.java (hunk 2 above)
Comment (miguelandres) [reply remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/view/RemixerFragment.java]: Done

**Pull request merged**